schemas/dab.draft.schema.json

```diff
@@ -261,6 +261,28 @@
             }
           }
         },
+        "logger-level": {
```

Comment:
We should stick with the originally proposed name `log-level`. 

To address Ani's comment about naming not colliding with built in types: you can have the publicly exposed JSON Property name be `log-level` and the actual record member name be `LoggerLevel`